src/dbt_score/config.py

```diff
@@ -67,6 +69,8 @@ def __init__(self) -> None:
         self.rules_config: dict[str, RuleConfig] = {}
         self.config_file: Path | None = None
         self.badge_config: BadgeConfig = BadgeConfig()
+        self.fail_project_under: float = 0.0
+        self.fail_any_model_under: float = 0.0
```

Comment:
I would be in favour of a breaking change and have reasonable defaults here, e.g. `5.0`